tools/clang/test/CodeGenSPIRV/spirv.legal.tbuffer.hlsl

```diff
@@ -19,8 +19,8 @@ struct S {
     float4 f;
 };
 
-// CHECK: %myCBuffer = OpVariable %_ptr_Uniform_type_TextureBuffer_S Uniform
-TextureBuffer<S>         myCBuffer;
+// CHECK: %myTBuffer = OpVariable %_ptr_Uniform_type_TextureBuffer_S Uniform
```

Comment:
lol, I should have caught this in the previous reviews! oops

Reply:
Haha. No worries. :)